source/stylist/source.py

```diff
@@ -461,34 +531,37 @@ def __init__(self,
 class SourceFactory(object):
     """
     Manages the handling of source file. Knows what chains of objects are
-    needed to each file extension.
-    """
-    # It is hard to lay down hard and fast rules about what should go in the
-    # _extension_map list. Languages standards do not generally specify an
-    # extension since it is an OS level concept.
-    #
-    # The default list should be kept short and the two goals should be
-    # "correctness" and "commodity".
-    #
-    # Correctness means not including a slew of extensions just because they
-    # happen to be used. Stick to ones which are "correct" in that they are
-    # well thought out.
-    #
-    # For instance it is not uncommon for people to use all sorts of extensions
-    # for Fortran which encodes the version of the language they are using.
-    # This is a poor idea as it leads to a proliferation of extension for no
-    # obvious gain. What's more a file with a ".f03" extension may be coded
-    # using only Fortran 95 features.
-    #
-    # Instead we follow the convention that ".f90" means "free format" source
-    # while ".f" continues to mean "fixed format".
-    #
-    # On the other hand commodity means including the most commonly used
-    # extensions. That is why both ".cc" and ".cpp" are listed for C++ source.
-    #
-    # If your particular application needs to support some odd-ball extensions
-    # then it can use the add_extension() call.
-    #
+    needed to handle each file extension.
+
+    It is hard to lay down hard and fast rules about what should go in the
+    default list of extensions. Language standards do not generally specify an
+    extension since that is an OS level concept. Some filesystems do not use
+    extensions at all.
+
+    The default list is kept short and the two goals are "correctness" and
+    "typicallity".
```

Comment:
I think one "l" in "typicality".

Reply:
It's a made up word so we can probably argue for whatever we want but I have a real blind spot for whether consenants should be doubled or not.